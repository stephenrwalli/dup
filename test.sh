# A simple test script to ensure dup works. Includes an empty file.

echo "We are expecting the following numbers:"
cat test_expected.txt
echo " "
echo "Go is not required to maintain maps in the same order,"
echo "so different runs can produce results ordered differently."
echo "Test run output:"
dup testdata/* 


