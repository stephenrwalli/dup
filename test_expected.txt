3	hello
3	bye
2	cat
2	dog
